build.gradle

```diff
@@ -1,13 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
+    ext {
+        compose_version = '1.2.0-alpha04'
+        kotlin_version = '1.6.10'
+    }
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:7.0.0-rc01'
```

Comment:
Sorry, one more thing I just noticed. I think we have 7.1.2 now stable

Reply:
Done.